src/utils/holidays.js

```diff
@@ -0,0 +1,14 @@
+const emojis = new Map([
+  [`New Year's Day`, ":baby-new-year:"],
```

Comment:
this looks like it could be normal double quotes?

```suggestion
  ["New Year's Day", ":baby-new-year:"],
```